src/p5.dance.js

```diff
@@ -81,6 +81,7 @@ module.exports = class DanceParty {
 
     this.world.bg_effect = null;
     this.world.fg_effect = null;
+    this.world.aiBlockSuccess = false; // This helps keeps track of whether the AI block successfully executed or not.
```

Comment:
nit: the rest of this file seems to use comments above the line in question, so I think best to do it that way here, too.

Reply:
I actually just removed this initialization.